gemini/sample-apps/e2e-gen-ai-app-starter-pack/app/server.py

```diff
@@ -35,7 +35,7 @@
 # Or choose one of the following pattern chains to test by uncommenting it:
 
 # Custom RAG QA
-# from app.patterns.custom_rag_qa.chain import chain
+# from app.patterns.agent_builder_search.chain import chain
```

Comment:
I wouldn't do this! 

Perhaps we can suggest people to look at the pattern folder to see more patterns!